src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const obj = {};
```

Comment:
The variable name 'obj' is not descriptive. Consider using a more descriptive name like 'styleObject' to improve code readability.